rel-sign.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
```

Comment:
I've run it through `shellcheck` and it found some issues which I think would harden the script when fixed.

Rather than enumerating each `shellcheck` issue, it may just be quicker to run it in your editor.

The first issue relates to the use of `/bin/sh` vs `/bin/bash`.  Since we'd like to use Bash-specific builtins like `[[ ]]`, I think we can just change the shebang line to `#!/bin/bash`.

Once we have got that, the rest of the `shellcheck` warnings can be fixed accordingly.

Reply:
Thanks, fixed bash thing and looking at the shellcheck (actually, didn't use it before - shell scripting is not a thing I'm powerful in, have a lot to learn :) ).